app/commands/mvcha.js

```diff
@@ -0,0 +1,70 @@
+module.exports = {
+    name: "mvcha",
+    description: "Rename/Remove a channel to category",
```

Comment:
what's the diff between remove and move? 
could u explain @MutlaqAldhbuiub

Reply:
Thank you so much @grepaziz. 
**This is a stuipd mistake, I'm so sorry about it.** 
This should be "Move" not "Remove". 
Because there's a command to remove channels "rmcha".
This command "mvcha" moves the channel from one category to another category and also you don't need to move this channel, you could only rename the channel {optional parameter}.
This idea comes from "mv" command from GNU/LINUX

Again Thank you so much Aziz <3